src/mono/mono/metadata/assembly.c

```diff
@@ -1769,6 +1771,26 @@ netcore_load_reference (MonoAssemblyName *aname, MonoAssemblyLoadContext *alc, M
 	return reference;
 }
 
+static MonoImage *
+netcore_open_from_satellite_bundle (MonoAssemblyLoadContext *alc, const char *filename, MonoImageOpenStatus *status, gboolean refonly, const char *culture)
```

Comment:
Nit: I wouldn't call this `netcore_` since there isn't a legacy equivalent. `open_from_satellite_bundle` seems fine on its own.